template/src/stories/Card.stories.tsx

```diff
@@ -1,7 +1,6 @@
-import React from 'react';
 import type { ComponentMeta, ComponentStory } from '@storybook/react';
 
-import { Card } from './Card';
+import { Card } from 'modules/ui';
 
 export default {
     title: 'UI/Card',
```

Comment:
I would keep the stories alongside the component:
Its recomeded to do that: https://storybook.js.org/docs/react/writing-stories/introduction

Reply:
Agreed

Reply:
ok, makes sense